### DISCUSSION TITLE
No longer works on 7.0 or above - Giving up

### DISCUSSION THREAD

**Discussion by Webslug** [cefsharp/CefSharp#4807] [General]
### Is there an existing issue for this?

- [X] I have searched both open/closed issues, no issue already exists.

### CefSharp Version

124.3.80

### Operating System

Windows 10

### Architecture

x64

### .Net Version

7.0

### Implementation

WinForms

### Reproduction Steps

![cefsharp](https://github.com/cefsharp/CefSharp/assets/7551638/caa11ec9-f911-47c8-a0a1-ce8c385aa58c)

 using CefSharp;
using CefSharp.WinForms;

namespace cefsharp
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            InitializeChromium();
        }

        private void Form1_FormClosing(object sender, FormClosingEventArgs e)
        {
            Cef.Shutdown();
            base.OnFormClosing(e);
        }

        private void InitializeChromium()
        {
            chromiumWebBrowser1 = new ChromiumWebBrowser("about:blank")
            {
                Dock = DockStyle.Fill
            };
            this.Controls.Add(chromiumWebBrowser1);

            chromiumWebBrowser1.LoadingStateChanged += chromiumWebBrowser1_LoadingStateChanged;
        }

        private void chromiumWebBrowser1_LoadingStateChanged(object sender, LoadingStateChangedEventArgs e)
        {
            if (!e.IsLoading)
            {
                // Once the page has finished loading, execute JavaScript to display "Hello, World!" in the browser
                chromiumWebBrowser1.ExecuteScriptAsync("document.body.innerHTML = '<h1>Hello, World!</h1>';");
            }
        }
    }
}

using CefSharp.WinForms;
using System.Runtime.ConstrainedExecution;
using CefSharp;
using CefSharp.WinForms;

namespace cefsharp
{
    internal static class Program
    {
        /// <summary>
        ///  The main entry point for the application.
        /// </summary>
        [STAThread]
        static void Main()
        {
            // Set up CefSharp settings
            CefSettings settings = new CefSettings();
            // Initialize CefSharp with the provided settings
            Cef.Initialize(settings);

            ApplicationConfiguration.Initialize();
            Application.Run(new Form1());
        }
    }
}

### Expected behavior

I wanted a working web browser.  CefSharp just does not work in Visual Studio anymore.

### Actual behavior

Does not run or compile.  

### Regression?

I tried on version 7.0 and version 8.0

### Known Workarounds

None found

### Does this problem also occur in the CEF Sample Application

Yes using WPF/OffScreen command line args

### Other information

using CefSharp;
using CefSharp.WinForms;

namespace cefsharp
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            InitializeChromium();
        }

        private void Form1_FormClosing(object sender, FormClosingEventArgs e)
        {
            Cef.Shutdown();
            base.OnFormClosing(e);
        }

        private void InitializeChromium()
        {
            chromiumWebBrowser1 = new ChromiumWebBrowser("about:blank")
            {
                Dock = DockStyle.Fill
            };
            this.Controls.Add(chromiumWebBrowser1);

            chromiumWebBrowser1.LoadingStateChanged += chromiumWebBrowser1_LoadingStateChanged;
        }

        private void chromiumWebBrowser1_LoadingStateChanged(object sender, LoadingStateChangedEventArgs e)
        {
            if (!e.IsLoading)
            {
                // Once the page has finished loading, execute JavaScript to display "Hello, World!" in the browser
                chromiumWebBrowser1.ExecuteScriptAsync("document.body.innerHTML = '<h1>Hello, World!</h1>';");
            }
        }
    }
}

**Comment by Webslug** [comment]
Not much support.  Well thanks anyway.

**Reply by amaitland** [reply]
> Not much support. Well thanks anyway.

Please show some patience. It's not practical for me to respond to non sponsors within 24hrs

> I tried on version 7.0 and version 8.0

Are you using the `NETCore` nuget packages?

Follow the instructions at https://github.com/cefsharp/CefSharp/wiki/Quick-Start-For-MS-.Net-5.0-or-greater

Testing with https://github.com/cefsharp/CefSharp.MinimalExample and `.Net 8` is working as expected.

**Reply by Webslug** [reply]
I apologize, yes that's the one which actually installed and was functional.  The example mini chrome works perfectly fine.  

I'm just so confused.

**Reply by amaitland** [reply]
Did you follow the quick start?

Did you specify a RuntimeIdentifier?
